subprojects/core/core.gradle.kts

```diff
@@ -72,6 +73,7 @@ dependencies {
     testImplementation(project(":plugins"))
     testImplementation(project(":testingBase"))
     testImplementation(project(":platformNative"))
+    testImplementation(project(":files"))
```

Comment:
I don't think it makes sense to have `:files` in `implementation` and `testImplementation`, does it? I would expect that everything in `implementation` is also in `testImplementation`.

Reply:
Done.